aw_datastore/storages/memory.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import sys
+import copy
+from datetime import datetime
+
+from aw_core.models import Event
+
+from . import logger
+from .abstract import AbstractStorage
+
+logger = logger.getChild("memory")
```

Comment:
Didn't see that it was "getChild".

Fixed and made it consistent between storages